release-notes/search-processor.release-notes-2.5.0.md

```diff
@@ -0,0 +1,38 @@
+Compatible with OpenSearch (2.5.0).
+
+### Features
+
+* Add a Docker quickstart script for demo / test ([#25](https://github.com/opensearch-project/search-processor/pull/25))
+* Configure Mend for GitHub.com ([#37](https://github.com/opensearch-project/search-processor/pull/37))
```

Comment:
Should this be Maintenance?

Reply:
According to [RELEASE_NOTES.md](https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md), `Maintenance` is "A change to add support for new versions of OpenSearch or OpenSearch Dashboards from upstream."

#37 is to configure settings before Mend starts scanning your repository for security vulnerabilities. Does it make more sense to fall into `Infrastructure`: A change to infrastructure, testing, CI/CD, pipelines?

Reply:
Infrastructure or Security

Reply:
Thanks! change https://github.com/opensearch-project/search-processor/pull/37 to label with `infra`